php/Laravel42/app/routes.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+|--------------------------------------------------------------------------
+| Application Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you can register all of the routes for an application.
+| It's a breeze. Simply tell Laravel the URIs it should respond to
+| and give it the Closure to execute when that URI is requested.
+|
+*/
+
+Route::get('/', ['uses' => 'HomeController@showWelcome', 'as' => 'simple_route_name']);
+Route::get('/distributed-tracing', 'HomeController@distributedTracing');
```

Comment:
I get an Apache-404 page when I try to visit this URL. I'm wondering if we're missing the Apache config for mod_rewrite? We should probably mention this route in the README, or was this just a left over test? :)

Reply:
Probably I left mod_rewrite config out when moving from the preconfigured image to the one based on `php` base images. Thanks for noticing. Looking at it